Bug fixes and small alterations (#115)
rostools/r-cubed-advanced: sessions/pipelines.qmd

<!-- TODO: Check how this works, and update next section -->

> While this `{targets}` pipeline work, we would not be able to

Reviewer: ```suggestion
While this `{targets}` pipeline works, we would not be able to
```

> automatically rerun out pipeline if our underlying data changes. To

Reviewer: ```suggestion
automatically re-run our pipeline if our underlying data changes. To
```

> track the actual data file we need to create a pipeline target of data

Reviewer: ```suggestion
track the actual data file we need to create a pipeline target of the data
```

> file. We can acomplish this using the argument `format = "file"` inside

Reviewer: ```suggestion
file. We can accomplish this by using the argument `format = "file"` inside
```

> `tar_target()` before loading the data using `{readr}`.

Reviewer: ```suggestion
the `tar_target()` before loading the data using `{readr}`.
```
